NimHM/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿string matches = "||||||||||||||||||||||||";//scoreboard
+string removedMatches;
+string inputNumber;//number given by player then converted to int "chosenNumber"
+int chosenNumber;
+int count = 24; // to keep check of remaining matches, removes from this every time an action is completed
+var randomGenerator = new Random(); //random number generator, later forced to remain within 1-4 which gives us numbers 1,2,3
+var numberOfTurns = 24;
+Console.WriteLine("Welcome to Nim!");
+
+ThreadStart:
+if
+    (count < 1)
+{
+    Console.WriteLine("You lose!");
+    Console.ReadKey();
+}
+
+Console.WriteLine("How many matches do you want to remove? (1,2,3)");
+    Console.WriteLine(matches + count+" Remaining matches");
+
+    inputNumber = Console.ReadLine();
+    chosenNumber = int.Parse(inputNumber);
+    bool ChosenNumber1 = chosenNumber < 2 && chosenNumber > 0; // boolian values that gives player the correct action when inputting 1,2,3
```

Comment:
you could also do `chosenNumber == 1` here. But I don't think that you would ever need the `ChosenNumber1` variable.